src/screens/ProfileScreen/elements/RenderItem.js

```diff
@@ -186,7 +191,7 @@ const Item = ({
           item={item}
           totalComment={getCountCommentWithChild(item)}
           totalVote={totalVote}
-          isSelf={true}
+          isSelf={false}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isSelf` prop is now set to `false`. If this change is intentional, ensure it doesn't affect other parts of the application that rely on this prop.
```diff
- isSelf={true}
+ isSelf={false}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->